style/themes/default-light.css

```diff
@@ -64,7 +64,7 @@
   color: #fff;
 }
 .sidebar-menu > li.header {
-  color: #4b646f;
+  color: rgb(173, 166, 156);
```

Comment:
In my opinion, the color you chose (`rgb(173, 166, 156)`) is perfect for the Darker theme, but I think It is too "yellowish" to be used with other themes.

My suggestion to Light theme is:
`#808a90` (or `rgb(128, 138, 144)`, if you want to use rgb() function).